responses/pl/HassLightSet.yaml

```diff
@@ -2,7 +2,7 @@ language: pl
 responses:
   intents:
     HassLightSet:
-      brightness: '{{ slots.name }} brightness set to {{ slots.brightness }}'
-      brightness_area: Brightness in {{ slots.area }} set to {{ slots.brightness }}
-      color: '{{ slots.name }} color set to {{ slots.color }}'
-      color_area: Color in {{ slots.area }} set to {{ slots.color }}
+      brightness: Ustawiono jasność {{ slots.name }} na {{ slots.brightness }}
```

Comment:
The reason is that it works like that because you use `20%`. If you replace this value to `20 %` wit will work differently. Now is quite hard discover that you provide unit or not 😀

Reply:
I don't know how to discover when a value contains a unit and when not, can you suggest a solution?

Reply:
https://github.com/home-assistant/intents/issues/811